distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -475,18 +475,18 @@ type Table
        > Example
          Order the table by the column "alpha" in ascending order.
 
-             table.order_by (Sort_Column_Selector.By_Name [Sort_Column.Name "alpha"])
+             table.order_by (Sort_Column_Selector.By_Name ["alpha"])
 
        > Example
          Order the table by the second column in ascending order. In case of any
          ties, break them based on the 7th column from the end of the table in
          descending order.
 
-             table.order_by (Sort_Column_Selector.By_Index [Sort_Column.Index 1, Sort_Column.Index -7 Sort_Direction.Descending])
+             table.order_by (Sort_Column_Selector.By_Index [1, Sort_Column.Index -7 Sort_Direction.Descending])
        > Example
          Sorting `table` in ascending order by the value in column `'Quantity'`.
 
-             table.order_by (Sort_Column_Selector.By_Name [Sort_Column.Name 'Quantity'])
+             table.order_by (Sort_Column_Selector.By_Name ['Quantity'])
```

Comment:
I noticed that `Database` and `Table` examples for `order_by` differ slightly - here we have `'Quantity'` and there we have `"price"`.

It's not a big deal, but I think we need to decide if we should enforce keeping the same docstrings between Table and Database or not.

@jdunkerley do you think I should unify them in this PR? Any suggestions which one to choose? I guess both examples are rather the same so it doesn't matter as long as we pick one.